sky/cli.py

```diff
@@ -1433,9 +1439,30 @@ def start(clusters: Tuple[str], yes: bool, idle_minutes_to_autostop: int,
       \b
       # Restart multiple clusters.
       sky start cluster1 cluster2
+      \b
+      # Restart all clusters.
+      sky start -a
 
     """
     to_start = []
+
+    if not len(clusters) and not all:
```

Comment:
style nit: use `not clusters` instead of `not len(clusters)` to decide whether the list is empty.